src/openms/source/FORMAT/HANDLERS/MzIdentMLHandler.cpp

```diff
@@ -743,26 +743,30 @@ namespace OpenMS
             {
               String pevid =  "PEV_" + String(UniqueIdGenerator::getUniqueId());
               String dBSequence_ref = String(sen_ids.find(pe->getProteinAccession())->second);
-              String idec(boost::lexical_cast<std::string>((String(jt->getMetaValue("target_decoy"))).hasSubstring("decoy")));
+              String idec;
+              if (jt->metaValueExists("target_decoy"))
+              {
+                idec = String(boost::lexical_cast<std::string>((String(jt->getMetaValue("target_decoy"))).hasSubstring("decoy")));
+              }
 
               String e;
-              e += "\t<PeptideEvidence id=\"" + pevid + "\" peptide_ref=\"" + pepid + "\" dBSequence_ref=\"" + dBSequence_ref;
+              e += "\t<PeptideEvidence id=\"" + pevid + "\" peptide_ref=\"" + pepid + "\" dBSequence_ref=\"" + dBSequence_ref + "\"";
```

Comment:
No, as you can see, all tests passed, except the mac build stuff.
I changed escaping to be line complete in that scope, means where there is an opening "\"" there should be a closing "\"" in the same line.